iotdbfrontend/src/components/Dashboard.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import LogInForm from './LogInForm';
+import LogOutButton from './LogOutButton'
+import Cookies from 'js-cookie';
+
+
+const Dashboard = () => {
+    var jwtExists = false
+    console.log(Cookies.get('logged_in'))
+    if (Cookies.get('logged_in') == 'true') {
+        jwtExists = true
+    }
```

Comment:
Why not
```js
const jwtExists = (Cookies.get('logged_in') === 'true')
```

Reply:
Much cleaner. I've actually gone further and done this:
```suggestion
    const [loggedIn, setLoggedIn] = useState(Cookies.get('logged_in') == 'true')
```